examples/default/Makefile

```diff
@@ -34,6 +34,12 @@ USEMODULE += ps
 # include and auto-initialize all available sensors
 USEMODULE += saul_default
 
+USEMODULE += random
+# Using this module might print warnings about unsatisfied features such as HWRNG
+# and ADC, depending on your hardware resources. That must not necessarily mean
+# that the seed is bad. Have a look at the random_seed documentation.
+# USEMODULE += random_seed
```

Comment:
This I really don't like. We must find a way so that the fitting/optimal modules for the seed module are included in a way that doesn't produce compile time warnings! I mean this is kind the idea for the fallback mechanism proposed here...